### DISCUSSION TITLE
University Lab Exercises using Digital

### DISCUSSION THREAD

**Discussion by wmuece** [hneemann/Digital#1266] [Show and tell]
A Digital Design Lab has been developed and employed the last eight semesters at Western Michigan University, based on Digital.  It consists of 18 exercises that have been used in our large lecture freshman ECE 2500 Digital Logic course.  The Digital Design Lab is available in two forms: 1) As part of the self-grading zybook textbook from the zyBook publishing company, zybook.com, with the grading done by the zyBook software.  Ask for my version.  2) a non-self grading pdf copy of the lab exercises that is linked here, which shows the entire sequence of exercises, with links:  https://webwriters.com/ece2500/DigitalDesignLab.pdf
<img width="1227" alt="Screenshot 2024-02-18 at 3 30 52 PM" src="https://github.com/hneemann/Digital/assets/1489640/5cd95d88-32ce-4fd6-a061-2a202008633f">

**Comment by TheWildJarvi** [comment]
So happy to see Universities starting to use Digital more and more. Logisim and its children are so slow/outdated/bulky

**Comment by wmuece** [comment]
Yes we had a good experience with it.  Just needs a more direct path to the hardware boards.

Sent from my iPhone ... Prof. J.

On Apr 5, 2024, at 5:53 PM, TheWildJarvi ***@***.***> wrote:

﻿

Attention: This email is from outside Western Michigan University. Use caution when opening links and attachments.

So happy to see Universities starting to use Digital more and more. Logisim and its children are so slow/outdated/bulky
